barretenberg/cpp/src/barretenberg/honk/transcript/transcript.test.cpp

```diff
@@ -185,6 +185,34 @@ TEST_F(UltraTranscriptTests, VerifierManifestConsistency)
     }
 }
 
+/**
+ * @brief Check that multiple challenges can be generated and sanity check
+ * @details We generate 6 challenges that are each 128 bits, and check that they are not 0.
+ *
+ */
+TEST_F(UltraTranscriptTests, ChallengeGenerationTest)
```

Comment:
Test looks good but in general we don't want to have stuff printing in tests. It's fine to leave the prints on the ASSERTs but go ahead and remove the `info`s

Reply:
other tests in this fixture also have infos, so I assumed it would be ok.

Reply:
Totally understandable, but those tests shouldn't be printing either